azurerm/resource_arm_servicebus_topic_test.go

```diff
@@ -165,7 +165,7 @@ func TestAccAzureRMServiceBusTopic_enablePartitioningPremium(t *testing.T) {
 			{
 				Config: postConfig,
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(resourceName, "enable_partitioning", "true"),
+					resource.TestCheckResourceAttr(resourceName, "enable_partitioning", "false"),
```

Comment:
Yo'd better validate the behavior  if `enable_partitioning` is true.

Reply:
This isn't possible due to the API behaviour change/limitation